<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">padding: 10 px;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="Word_write_textedit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>571</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #bcbeb3;
QTextEdit { line-height: 15; };
border: none;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Start_new_test_button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333438</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="English_space_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>842</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="down_gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="English_z_button">
       <property name="text">
        <string>z</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="English_x_button">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="English_c_button">
       <property name="text">
        <string>c</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="English_v_button">
       <property name="text">
        <string>v</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="English_b_button">
       <property name="text">
        <string>b</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="English_n_button">
       <property name="text">
        <string>n</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="English_m_button">
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="English_comma_button">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="English_dot_button">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>942</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mid_gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="English_a_button">
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="English_s_button">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="English_d_button">
       <property name="text">
        <string>d</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="English_f_button">
       <property name="text">
        <string>f</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="English_g_button">
       <property name="text">
        <string>g</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="English_h_button">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="English_j_button">
       <property name="text">
        <string>j</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="English_k_button">
       <property name="text">
        <string>k</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="English_l_button">
       <property name="text">
        <string>l</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="English_CAD_button">
       <property name="text">
        <string>;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QPushButton" name="English_quotes_button">
       <property name="text">
        <string>'</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>1028</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="up_gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="English_q_button">
       <property name="styleSheet">
        <string notr="true">height 10 px;</string>
       </property>
       <property name="text">
        <string>q</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="English_w_button">
       <property name="text">
        <string>w</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="English_e_button">
       <property name="text">
        <string>e</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="English_r_button">
       <property name="text">
        <string>r</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="English_t_button">
       <property name="text">
        <string>t</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="English_y_button">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="English_u_button">
       <property name="text">
        <string>u</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="English_i_button">
       <property name="text">
        <string>i</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="English_o_button">
       <property name="text">
        <string>o</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="English_p_button">
       <property name="text">
        <string>p</string>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QPushButton" name="English_array_left_button">
       <property name="text">
        <string>[</string>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QPushButton" name="English_array_right_button">
       <property name="text">
        <string>]</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Belarusian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>German</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>French</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arabic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chinese</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hebrew</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
